handlers/handlers_test.go

```diff
@@ -29,70 +25,6 @@ func TestOKHandler(t *testing.T) {
 	require.Equal(rr.Body.String(), "OK")
 }
 
-func TestSegmentCallback(t *testing.T) {
```

Comment:
Just to clarify - we're removing this test instead of updating it because now that transcode is not stubbed out, we don't have a good way to stub the MistProcLivepeer exec command so making this test pass would be tricky?

Reply:
Yes